File: LV2/이진 변환 반복하기/solution.py
def solution(s):
    answer = []
    countZero = 0
    num = 0
    
    while True :
        count = s.count('0')
        countZero += count
        
        leng = len(s.replace('0',''))
        num+=1
        if leng == 1:
            break
        s=format(leng,'b')
        
    answer = [num,countZero]
    return answer

#https://school.programmers.co.kr/learn/courses/30/lessons/70129?language=python3

File: LV2/이진 변환 반복하기/solution.js
function solution(s) {
    var answer = [];
    var countZero = 0;
    var num = 0
    while (true) {
        var count = s.split('0').length-1;
        countZero+=count
        
        var len = s.replaceAll('0','').length;
        num++
        if(len=="1") break;
        s=len.toString(2)
    }
    answer = [num,countZero]
    return answer;
}

// https://school.programmers.co.kr/learn/courses/30/lessons/70129